simple.zig

```diff
@@ -0,0 +1,42 @@
+const std = @import("std");
+const WordMap = std.StringHashMap(u32);
+
+pub fn main() !void {
+    var arena = std.heap.ArenaAllocator.init(std.heap.page_allocator);
```

Comment:
Wow, this is the first time I've seen non-trivial Zig code -- allocation is explicit! I should look into this language further. Looks really useful as a C replacement for embedded development.